controls/nist_ocp4.yml

```diff
@@ -15699,13 +15698,15 @@ controls:
     applicable to the configuration of OpenShift.
   rules: []
 - id: SI-4(22)
-  status: pending
+  status: automated
   notes: |-
-    A control response for SI-4(20) is planned. Engineering progress can be
-    tracked via:
+    The customer is responsible for ensuring that NetworkPolicies
+    are configured in all non-openshift namespaces to allow traffic
+    only in the namespace and not across namespaces[1]
```

Comment:
I think we should reword this a little and explain that while OCP doesn't provide a way to audit the services per se, explicitly setting NetworkPolicies would help. They might still need another way to make the service approved from their end (inventory, trusting only certain CAs...)

Reply:
I am wondering how is `inventory, trusting only certain CAs` helping this case?

Reply:
So inventory is just a list of the approved services. A way to match the network policies to the outside. Trusting only certain CAs is probably only helpful for routes (see e.g. https://docs.openshift.com/container-platform/4.10/networking/routes/secured-routes.html) so they check that only the CA certs that the org should be trusting are set for routes. In-cluster communication is done using self-signed certs anyway.